### ISSUE TITLE
No access to the marketplace when remote is set to false

### ISSUE THREAD

**Issue by lolodomo** [openhab/openhab-webui#1245]
For users who decided to rely only on an offline distribution and so who setup openHAB like this:
```

# Access Remote Add-on Repository
# Defines whether the remote openHAB add-on repository should be used for browsing and installing add-ons. (default is true)
#
remote = false
```

I assume MainUI should not show marketplace in this place but only bindings present in the kar file.

**Comment by kaikreuzer** [comment]
I agree, that's what such users would expect.
Question is if the download of the icons for official add-ons should also be deactivated in such a case?

**Comment by ghys** [comment]
It's not right to put the logic in the UI problem in my view, the community marketplace add-on service should check this flag and return an empty list of add-ons if remote access is disabled.
Ideally it would be the same for icons, return the URL to display (or not) in `imageLink` instead of the logic being in the UI.

**Comment by kaikreuzer** [comment]
I agree, so let's move this to openhab-core then (and not include it in 3.2!)
